stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>"at least 4 times"</p><p> </p><p>In no way obligatory, but many style guides suggest using words for numbers less than 10, which reads more naturally to me.</p><p> </p><p>https://www.bbc.co.uk/newsstyleguide/numbers/</p><p>https://www.theguardian.com/guardian-observer-style-guide-n</p><p> </p><p>There are a few cases below as well in case you do decide to make this change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>